conduit/src/main/java/io/rtr/conduit/amqp/impl/AMQPConsumerBuilder.java

```diff
@@ -26,7 +26,7 @@
     private boolean automaticRecoveryEnabled = true;
     private int retryThreshold = 10;
     private boolean poisonQueueEnabled = true;
-    private int prefetchCount = 1;
+    private int prefetchCount = 250;
```

Comment:
Sorry to jump in on a draft PR, but this is scary as a default haha.  Why not use the builder method to customize it for your use case?